deployment/bare_metal/.env.dist

```diff
@@ -26,7 +26,7 @@ COMMUNITY_REDEEM_CONTRIBUTION_URL=https://stage1.gradido.net/redeem/CL-{code}
 COMMUNITY_DESCRIPTION="Gradido Development Stage1 Test Community"
 
 # backend
-BACKEND_CONFIG_VERSION=v10.2022-09-20
+CONFIG_VERSION=v11.2022-10-27
```

Comment:
was the replaced BACKEND_CONFIG_VERSION-property never used in the baremetal-scripts?
If it is in use, where are the changes about this exchange to CONFIG_VERSION?

Reply:
Good catch. Must be `BACKEND_CONFIG_VERSION` here. As in the `backend/.env.template` it states:
`CONFIG_VERSION=$BACKEND_CONFIG_VERSION`